### PR TITLE
Add Game Datasets

### PR DESCRIPTION
**Pull request by leomaurodesenv**
https://github.com/leomaurodesenv/game-datasets

This repository serves as a guide for anyone who wants to work with Data Mining or Artificial Intelligence applied in digital games, with several tools and datasets.

### By submitting this pull request I confirm I've read and complied with the below requirements 🖖

**Please read it multiple times. I spent a lot of time on these guidelines and most people miss a lot.**

## Requirements for your pull request

- **You have to review at least 2 other [open pull requests](https://github.com/sindresorhus/awesome/pulls?q=is%3Apr+is%3Aopen).** Try to prioritize unreviewed PRs, but you can also add more comments to reviewed PRs. Go through the below list when reviewing. This requirement is meant to help make the Awesome project self-sustaining. Comment here which PRs you reviewed. You're expected to put a good effort into this and to be thorough. Look at previous PR reviews for inspiration.
- I have read and understood the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/master/create-list.md).
- This pull request has a descriptive title in the format `Add Name of List` (Example: `Add Swift`), not `Update readme.md` or `Add awesome list`.
- The entry in the Awesome list should:
	- Include a short description about the project/theme of the list. **It should not describe the list itself.**<br>Example: `- [Fish](…) - User-friendly shell.`, not `- [Fish](…) - Resources for Fish.`.
	- Be added at the bottom of the appropriate category.
- The list I'm submitting complies with the below requirements.

## Requirements for your Awesome list

- **Has been around for at least 30 days.**<br>That means 30 days from either the first real commit or when it was open-sourced. Whatever is most recent.
- It's the result of hard work and the best I could possibly produce.
- Non-generated Markdown file in a GitHub repo.
- **Includes a succinct description of the project/theme at the top of the readme.** [(Example)](https://github.com/willempienaar/awesome-quantified-self)
- The repo should have `awesome-list` & `awesome` as [GitHub topics](https://help.github.com/articles/about-topics). I encourage you to add more relevant topics.
- Not a duplicate.
- Only has awesome items. Awesome lists are curations of the best, not everything.
- Includes a project logo/illustration whenever possible.
	- Either centered, fullwidth, or placed at the top-right of the readme. [(Example)](https://github.com/sindresorhus/awesome-electron)
	- The image should link to the project website or any relevant website.
	- The image should be high-DPI. Set it to maximum half the width of the original image.
- Entries have a description, unless the title is descriptive enough by itself. It rarely is though.
- Includes the [Awesome badge](https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge).
	- Should be placed on the right side of the readme heading.
		- Can be placed centered if the list has a centered graphics header.
	- Should link back to this list.
- Has a Table of Contents section.
	- Should be named `Contents`, not `Table of Contents`.
	- Should be the first section in the list.
	- Should only have one level of [nested lists](https://commonmark.org/help/tutorial/10-nestedLists.html), preferably none.
- Has an [appropriate license](https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license).
	- That means something like CC0, **not a code licence like MIT, BSD, Apache, etc.**
	- [WTFPL](http://www.wtfpl.net) and [Unlicense](http://unlicense.org) are not acceptable licenses.
	- If you use a license badge, it should be SVG, not PNG.
- Has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines).
	- The file should be named `contributing.md`. Casing is up to you.
- Has consistent formatting and proper spelling/grammar.
	- The link and description are separated by a dash. <br>Example: `- [AVA](…) - JavaScript test runner.`
	- The description starts with an uppercase character and ends with a period.
	- Consistent and correct naming. For example, `Node.js`, not `NodeJS` or `node.js`.
- Doesn't include a Travis badge.<br>You can still use Travis for list linting, but the badge has no value in the readme.

Go to the top and read it again.

**[Insert URL to the list here]**

**[Explain what this list is about and why it should be included here]**

### By submitting this pull request I confirm I've read and complied with the below requirements 🖖

**Please read it multiple times. I spent a lot of time on these guidelines and most people miss a lot.**

## Requirements for your pull request

- **You have to review at least 2 other [open pull requests](https://github.com/sindresorhus/awesome/pulls?q=is%3Apr+is%3Aopen).** Try to prioritize unreviewed PRs, but you can also add more comments to reviewed PRs. Go through the below list when reviewing. This requirement is meant to help make the Awesome project self-sustaining. Comment here which PRs you reviewed. You're expected to put a good effort into this and to be thorough. Look at previous PR reviews for inspiration.
- I have read and understood the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/master/create-list.md).
- This pull request has a descriptive title in the format `Add Name of List` (Example: `Add Swift`), not `Update readme.md` or `Add awesome list`.
- The entry in the Awesome list should:
	- Include a short description about the project/theme of the list. **It should not describe the list itself.**<br>Example: `- [Fish](…) - User-friendly shell.`, not `- [Fish](…) - Resources for Fish.`.
	- Be added at the bottom of the appropriate category.
- The list I'm submitting complies with the below requirements.

## Requirements for your Awesome list

- **Has been around for at least 30 days.**<br>That means 30 days from either the first real commit or when it was open-sourced. Whatever is most recent.
- It's the result of hard work and the best I could possibly produce.
- Non-generated Markdown file in a GitHub repo.
- **Includes a succinct description of the project/theme at the top of the readme.** [(Example)](https://github.com/willempienaar/awesome-quantified-self)
- The repo should have `awesome-list` & `awesome` as [GitHub topics](https://help.github.com/articles/about-topics). I encourage you to add more relevant topics.
- Not a duplicate.
- Only has awesome items. Awesome lists are curations of the best, not everything.
- Includes a project logo/illustration whenever possible.
	- Either centered, fullwidth, or placed at the top-right of the readme. [(Example)](https://github.com/sindresorhus/awesome-electron)
	- The image should link to the project website or any relevant website.
	- The image should be high-DPI. Set it to maximum half the width of the original image.
- Entries have a description, unless the title is descriptive enough by itself. It rarely is though.
- Includes the [Awesome badge](https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge).
	- Should be placed on the right side of the readme heading.
		- Can be placed centered if the list has a centered graphics header.
	- Should link back to this list.
- Has a Table of Contents section.
	- Should be named `Contents`, not `Table of Contents`.
	- Should be the first section in the list.
	- Should only have one level of [nested lists](https://commonmark.org/help/tutorial/10-nestedLists.html), preferably none.
- Has an [appropriate license](https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license).
	- That means something like CC0, **not a code licence like MIT, BSD, Apache, etc.**
	- [WTFPL](http://www.wtfpl.net) and [Unlicense](http://unlicense.org) are not acceptable licenses.
	- If you use a license badge, it should be SVG, not PNG.
- Has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines).
	- The file should be named `contributing.md`. Casing is up to you.
- Has consistent formatting and proper spelling/grammar.
	- The link and description are separated by a dash. <br>Example: `- [AVA](…) - JavaScript test runner.`
	- The description starts with an uppercase character and ends with a period.
	- Consistent and correct naming. For example, `Node.js`, not `NodeJS` or `node.js`.
- Doesn't include a Travis badge.<br>You can still use Travis for list linting, but the badge has no value in the readme.

Go to the top and read it again.

### CODE REVIEW

**Comment by mbasso** [comment]
> The entry in the Awesome list should be added at the bottom of the appropriate category.

It seems that this list has been added on the top of the `Gaming` category instead.

There are also some linting problems:

```
>awesome-lint https://github.com/leomaurodesenv/game-datasets
× Linting

  README.md:1:1
  ×   1:1    Missing or invalid Table of Contents                          remark-lint:awesome/toc
  ×   1:1    Missing newline character at end of file                      remark-lint:final-newline
  ×   1:1    Missing License section                                       remark-lint:awesome/license
  ×   1:31   Invalid badge source                                          remark-lint:awesome/badge
  ×  12:3    Invalid list item link URL                                    remark-lint:awesome/list-item
  ×  13:3    Invalid list item link URL                                    remark-lint:awesome/list-item
  ×  14:3    Invalid list item link URL                                    remark-lint:awesome/list-item
  ×  15:3    Invalid list item link URL                                    remark-lint:awesome/list-item
  ×  16:3    Invalid list item link URL                                    remark-lint:awesome/list-item
  ×  18:1    Rules should use ---                                          remark-lint:rule-style
  ×  21:180  Use two spaces for hard line breaks                           remark-lint:hard-break-spaces
  ×  30:1    Rules should use ---                                          remark-lint:rule-style
  ×  40:128  List item description must end with proper punctuation        remark-lint:awesome/list-item
  ×  48:112  List item description must end with proper punctuation        remark-lint:awesome/list-item
  ×  55:111  List item description must end with proper punctuation        remark-lint:awesome/list-item
  ×  56:133  List item description must end with proper punctuation        remark-lint:awesome/list-item
  ×  61:49   List item link and description must be separated with a dash  remark-lint:awesome/list-item
  ×  62:86   List item description must end with proper punctuation        remark-lint:awesome/list-item
  ×  63:116  List item description must end with proper punctuation        remark-lint:awesome/list-item
  ×  64:120  List item description must end with proper punctuation        remark-lint:awesome/list-item
  ×  65:100  List item description must end with proper punctuation        remark-lint:awesome/list-item
  ×  66:3    Incorrect indentation before bullet: remove 1 space           remark-lint:list-item-bullet-indent
  ×  66:74   List item description must end with proper punctuation        remark-lint:awesome/list-item
  ×  71:110  List item description must end with proper punctuation        remark-lint:awesome/list-item
  ×  76:87   List item description must end with proper punctuation        remark-lint:awesome/list-item

  25 errors
```

[https://github.com/sindresorhus/awesome-lint](https://github.com/sindresorhus/awesome-lint)

**Comment by leomaurodesenv** [comment]
Thanks for the review.

> It seems that this list has been added on the top of the Gaming category instead.

Yes, because of the alphabetical order.   
But, I will fix putting in the bottom.   

I forgot the [awesome-lint](https://github.com/sindresorhus/awesome-lint).   
I updated the layout right now.   
`✔ Linting`

**Comment by leomaurodesenv** [comment]
I'm currently following #1546 and #1545.

**Comment by leomaurodesenv** [comment]
Following #1553.

**Comment by sindresorhus** [comment]
Thanks for making an Awesome list! 🙌

It looks like you didn't read the [guidelines](https://github.com/sindresorhus/awesome/blob/master/pull_request_template.md) closely enough. I noticed multiple things that are not followed. Try going through the list point for point to ensure you follow it. I spent a lot of time creating the guidelines so I wouldn't have to comment on common mistakes, and rather spend my time improving Awesome.

**Comment by leomaurodesenv** [comment]
Thank you @sindresorhus,
I will double check this weekend!

**Comment by leomaurodesenv** [comment]
Hello @sindresorhus,
I just finish of review all guideline and also update list texts. Thank you for your time.

File: readme.md [hunk 1]
```diff
@@ -358,7 +358,7 @@
 - [Construct 2](https://github.com/WebCreationClub/awesome-construct#readme) - Game engine.
 - [Gideros](https://github.com/stetso/awesome-gideros#readme) - Game engine.
 - [Minecraft](https://github.com/bs-community/awesome-minecraft#readme) - Sandbox video game.
-
+- [Game Datasets](https://github.com/leomaurodesenv/game-datasets) - Game datasets and tools for Artificial Intelligence.
```
Comment (sindresorhus) [inline readme.md]: The description should succinctly explain what game datasets are.

**Comment by sindresorhus** [comment]
There should be a description at the top of the list readme about what game sets and data mining is.

---

> If you use a license badge, it should be SVG, not PNG.

File: readme.md (hunk 1 above)
Comment (sindresorhus) [reply readme.md]: You're also missing the `#readme`.

Comment (leomaurodesenv) [reply readme.md]: > The description should succinctly explain what game datasets are.

What do you think "Tools for Artificial Intelligence in Games."?

**Comment by leomaurodesenv** [comment]
> There should be a description at the top of the list readme about what game sets and data mining is.

Done. This really changed the introduction section.

**Comment by sindresorhus** [comment]
> If you use a license badge, it should be SVG, not PNG.

This is not done.

File: readme.md [hunk 2]
```diff
@@ -358,6 +358,7 @@
 - [Construct 2](https://github.com/WebCreationClub/awesome-construct#readme) - Game engine.
 - [Gideros](https://github.com/stetso/awesome-gideros#readme) - Game engine.
 - [Minecraft](https://github.com/bs-community/awesome-minecraft#readme) - Sandbox video game.
+- [Game Datasets](https://github.com/leomaurodesenv/game-datasets#readme) - Tools for Artificial Intelligence in Games.
```
Comment (sindresorhus) [inline readme.md]: `Games` => `games` and I think the description could still be clearer. `Tools` should also not be there as the list contains more than just tools.

**Comment by leomaurodesenv** [comment]
> > If you use a license badge, it should be SVG, not PNG.
> 
> This is not done.

Now, it is done. Sorry for this mistake.

File: readme.md (hunk 2 above)
Comment (leomaurodesenv) [reply readme.md]: "Materials and datasets for Artificial Intelligence in games."

Comment (sindresorhus) [reply readme.md]: Yes, better.

**Comment by sindresorhus** [comment]
Tweet: https://twitter.com/awesome__re/status/1139789622093869056

**Pull request merged**